nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -17,7 +27,7 @@ with lib;
     # Automatically login as root.
     displayManager.slim = {
```

Comment:
I won't ask it of this PR, but slim is long abandoned. We  should get off of it sometime :)

Reply:
Yep, I raised this in https://github.com/NixOS/nixpkgs/pull/66313#issue-305402121 at the very end. Some people had strong opinions though in the past IIRC.